.cargo/config.toml

```diff
@@ -1,3 +1,6 @@
+[env]
+RUST_TEST_THREADS = "1"
```

Comment:
Is this needed because `DB_MANAGER` is a global and we cannot ensure the proper initialization if there a concurrent tests? Or is it because we want to reset all RocksDB instances in a test and we don't wanna implement the means for it?

Reply:
It's because we can have multiple tests that race over initialization and may require exclusive control over database lifecycle.